packages/datagateway-download/src/downloadApi.ts

```diff
@@ -177,6 +177,33 @@ export const fetchDownloads: (
     });
 };
 
+export const fetchAdminDownloads: (
+  settings: { facilityName: string; downloadApiUrl: string },
+  queryOffset?: string
+) => Promise<Download[]> = (
+  settings: { facilityName: string; downloadApiUrl: string },
+  queryOffset?: string
+) => {
+  return axios
+    .get<Download[]>(`${settings.downloadApiUrl}/admin/downloads`, {
+      params: {
+        sessionId: readSciGatewayToken().sessionId,
+        facilityName: settings.facilityName,
+        queryOffset: !queryOffset
+          ? 'where download.isDeleted = false'
+          : queryOffset,
+      },
+    })
+    .then((response) => {
+      return response.data;
+    })
+    .catch((error) => {
+      // Should it be handling ICAT error or TopCat?
```

Comment:
The `handleICATError` function does take into account TopCAT errors - it's just a helper function for all bits of the ICAT stack so remove this comment